.bazelrc

```diff
@@ -35,8 +35,8 @@ build --copt -DGRPC_BAZEL_BUILD
 # Bazel defaults to JDK9. This makes sure we run Nix provided JDK, and
 # avoid dreaded "Unrecognized VM option 'CompactStrings'" error
 build --host_javabase=@java_home//:javabase
-build --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
-build --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build --java_toolchain=@com_github_digital_asset_daml//bazel_tools:toolchain_jdk11
```

Comment:
`@bazel_tools//tools/jdk:toolchain_hostjdk11` does not exist, see the comments in the definition of `toolchain_jdk11` for more information.